Profile/ProfilePage.xaml

```diff
@@ -1,188 +1,203 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-             xmlns:material="clr-namespace:Material.Components.Maui;assembly=Material.Components.Maui"
              x:Class="OnboardingSystem.ProfilePage"
              Title=""
              Shell.NavBarIsVisible="False">
 
-    <ContentPage.Resources>
+        <ContentPage.Resources>
         <!-- Updated the style to use Material components correctly -->
         <Style x:Key="MaterialEntryStyle" TargetType="Entry">
             <Setter Property="FontSize" Value="18" />
             <Setter Property="BackgroundColor" Value="White" />
             <Setter Property="TextColor" Value="Black" />
         </Style>
 
+        <!-- Style for disabled entries -->
+        <Style x:Key="DisabledEntryStyle" TargetType="Entry">
+            <Setter Property="FontSize" Value="18" />
+            <Setter Property="BackgroundColor" Value="#E1E1E1" />
+            <Setter Property="TextColor" Value="Gray" />
+            <Setter Property="IsEnabled" Value="False" />
+        </Style>
+
         <!-- Style for the label in the profile page -->
         <Style x:Key="ProfileLabelStyle" TargetType="Label">
             <Setter Property="FontSize" Value="18" />
-            <Setter Property="TextColor" Value="White" />
+            <Setter Property="TextColor" Value="{AppThemeBinding Light=Black, Dark=White}" />
             <Setter Property="Margin" Value="0,20,0,10" />
         </Style>
 
         <Style x:Key="Entry_Shape" TargetType="RoundRectangle">
             <!--Entry Bar Label-->
             <Setter Property="CornerRadius" Value="20" />
+        </Style>

```

Comment:
Styles should not be called or hardcoded in any single instance. Define the styles in styles.xml file and call them in profile instead. Refer to Resources>styles.xml  @Slashy-01